java-app-jcache/pom.xml

```diff
@@ -1,273 +1,338 @@
 <?xml version='1.0' encoding='utf-8'?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>javaee-cafe-jcache</artifactId>
-  <groupId>javaee-cafe</groupId>
-  <version>1.0.0</version>
-  <packaging>war</packaging>
-  <name>${project.artifactId}</name>
-  <description>This is the basic Java EE application used in the demo.
-  It is a simple CRUD application. It uses Maven and Java EE 8 (JAX-RS, EJB, CDI, JSON-B, JSF, Bean Validation).
-  </description>
+    <artifactId>javaee-cafe-jcache</artifactId>
+    <groupId>javaee-cafe</groupId>
+    <version>1.0.0</version>
+    <packaging>war</packaging>
+    <name>${project.artifactId}</name>
+    <description>This is the basic Java EE application used in the demo.
+    It is a simple CRUD application. It uses Maven and Java EE 10 (Jakarta RESTful Web Services, Jakarta Enterprise Beans, Jakarta CDI, Jakarta JSON-B, Jakarta Faces, Jakarta Validation).
```

Comment:
`Java EE 10` => `Jakarta EE 10`?

Reply:
Renamed to `Jakarta EE` in commit [e1361eb](https://github.com/Azure-Samples/open-liberty-on-aks/pull/47/commits/e1361ebc66e4eed2eb56e852d4bce3efd987a29b).